examples/fuse_examples/imaging/ml_pipelines/mnist/run_mnist_pipeline.py

```diff
@@ -0,0 +1,112 @@
+from fuse.dl.cross_validation.pipeline import run
+from examples.fuse_examples.imaging.classification.mnist.run_mnist import run_train, run_infer, run_eval
+from funcs import create_dataset
+import os
+
+##########################################
+# Required Parameters
+##########################################
+num_repetitions = 1
+num_gpus_total = 3
+num_gpus_per_split = 1
+num_folds = 5
+num_folds_used = 5
+dataset_func = create_dataset
+train_func = run_train
+infer_func = run_infer
+eval_func = run_eval
+deterministic_mode = True
+
+# output paths:
+root_path = "results"
+paths = {
```

Comment:
Why don't you import paths and params from the original example?

Reply:
done now for the params. I still define params from scratch to avoid confusion